.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,19 @@
+name: NPM publish
+
+on:
+  release:
+    types: [created]
+
```

Comment:
I recommend adding https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch in case you need to manually trigger after a CI bug